design-reviews/recipes/async-design-reviews.md

```diff
@@ -0,0 +1,85 @@
+# Async Design Reviews
+
+## Goals
+
+Allow team members to review designs as their work schedule allows.
+
+## Impact
+
+This in turn results in the following benefits:
+
+- **Higher Participation & Accessability**. They do not need to be online and available at the same time as other to review
+- **Reduced Time Constraint**. Reviewers can spend longer than the duration of a single meeting to think through approach and provide feedback.
+
+## Measures
+
+The metrics and/or KPI's used for design reviews overall would still apply. See [root readme](../readme.md#Measures) for measures guidance.
+
+## Participation
+
+The participation should be same as any design review. See [root readme](../readme.md#Participation) for participation guidance.
+
+## Facilitation Guidance
+
+The concept is to have the design follow the same workflow as any code changes to implement story or task. Rather than code however, the artifacts being added or changed are markdown documents as well as any other supporting artifacts (prototypes, code samples, diagrams, etc).

```

Comment:
Maybe we need to make the recipe more specific to markdown? It was not my intent to discourage the use of Word and its team collaboration tools. However, i think the workflow is going to be different since its not happening over pull request. This was not part of what i had in mind for "other supporting artifacts". Thoughts? Maybe title the doc "Async Design Reviews using Markdown"?

Reply:
Got it! Since it is likely more common for teams to use the Markdown/PR workflow, I think keeping it exactly as you have is perfect. If necessary in the future we could add another workflow for reviews using Word or other collaboration tools (which, some times, may make it easier to open up the review of the doc/comments to wider audience than one that has access to the code repo).

Reply:
Sounds good. Would be good to get feedback from others on how common the word approach is as well. Maybe after some of that data we can decide if it make sense to qualify this content a bit more.